Backend/app/rider-platform/rider-app/Main/spec/Storage/FrfsTicket.yaml

```
@@ -54,8 +54,7 @@ FRFSSearch:
     merchantOperatingCityId: Id MerchantOperatingCity
     partnerOrgTransactionId: Maybe (Id PartnerOrgTransaction)
     partnerOrgId: Maybe (Id PartnerOrganization)
-    journeyLegInfo: Maybe JourneySearchData
-    journeyLegStatus: Maybe JourneyLegStatus
+    journeyLegInfo: Maybe JourneySearchData    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)